src/components/v5/frame/ColonyHome/partials/ReputationChart/partials/TeamActionsMenu.tsx

```diff
@@ -0,0 +1,118 @@
+import { useActionSidebarContext } from '~context/ActionSidebarContext/ActionSidebarContext.ts';
+import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
+import { tw } from '~utils/css/index.ts';
+import { formatText } from '~utils/intl.ts';
+import { ACTION_TYPE_FIELD_NAME } from '~v5/common/ActionSidebar/consts.ts';
+
+const displayName =
+  'v5.frame.ColonyHome.ReputationChart.partials.TeamActionsMenu';
+
+const MSG = defineMessages({
+  viewAllTeams: {
+    id: `${displayName}.viewAllTeams`,
+    defaultMessage: 'View all teams',
+  },
+  createTeam: {
+    id: `${displayName}.createTeam`,
+    defaultMessage: 'Create new team',
+  },
+  editTeam: {
+    id: `${displayName}.editTeam`,
+    defaultMessage: 'Edit team',
+  },
+});
+
+// @TODO maybe decouple some of this into a dropdown menu component?
+// eslint-disable-next-line max-len
+const dropdownItemClassName = tw`flex w-full items-center gap-2 rounded-s px-3 py-2 text-md text-gray-900 hover:font-medium md:hover:bg-gray-50`;
+
+const TeamActionsMenu = () => {
+  const {
+    colony: { name: colonyName },
+  } = useColonyContext();
+  const [isDropdownOpen, setIsDropdownOpen] = React.useState(false);
+  const { getTooltipProps, setTooltipRef, setTriggerRef, visible } =
+    usePopperTooltip({
+      placement: 'bottom-end',
+      trigger: ['click'],
+      visible: isDropdownOpen,
+      onVisibleChange: setIsDropdownOpen,
+      interactive: true,
+      closeOnOutsideClick: true,
+      offset: [0, 0],
+    });
+
+  const {
+    actionSidebarToggle: [, { toggleOn: toggleActionSidebarOn }],
+  } = useActionSidebarContext();
+
+  const handleCreateTeamClick = () => {
+    toggleActionSidebarOn({
+      [ACTION_TYPE_FIELD_NAME]: Action.CreateNewTeam,
+    });
+    setIsDropdownOpen(false);
+  };
+
+  const handleEditTeamClick = () => {
+    toggleActionSidebarOn({
+      [ACTION_TYPE_FIELD_NAME]: Action.EditExistingTeam,
+    });
+    setIsDropdownOpen(false);
+  };
+
+  return (
+    <>
+      <button
+        type="button"
+        ref={setTriggerRef}
+        className={clsx('hover:text-blue-400', {
+          'text-gray-600': !visible,
+          'text-blue-400': visible,
+        })}
+      >
+        <DotsThreeVertical size={18} />
+      </button>
+      {visible && (
+        <div
+          ref={setTooltipRef}
+          {...getTooltipProps()}
+          className="z-dropdown w-full px-6 sm:w-auto sm:px-0"
+        >
+          <div className="flex w-full flex-col rounded-lg border border-gray-200 bg-base-white px-3 py-4 shadow-default sm:w-[14.25rem]">
+            <Link to={`/${colonyName}/teams`} className={dropdownItemClassName}>
+              <Eye size={18} />
+              <span>{formatText(MSG.viewAllTeams)}</span>
+            </Link>
+            <button
+              type="button"
+              className={dropdownItemClassName}
+              onClick={handleCreateTeamClick}
+            >
+              <Lock size={18} />
```

Comment:
This needs to use the `LockKey` icon based on Figma:

```js
<LockKey size={18} />
```

![Screenshot 2024-09-18 at 19 59 35](https://github.com/user-attachments/assets/71162afa-b8b0-4d47-9a0e-b412503e3ae2)